src/main/pg/gps.c

```diff
@@ -44,6 +44,7 @@ PG_RESET_TEMPLATE(gpsConfig_t, gpsConfig,
     .gps_use_3d_speed = false,
     .sbas_integrity = false,
     .gps_ublox_utc_standard = UBLOX_UTC_STANDARD_AUTO,
+    .nmeaCustomCommands = "",
```

Comment:
is this a fix for a specific bug that's been reported?

Reply:
Agree. Moved to another PR